content/en/architecture/component-overview.md

```diff
@@ -14,20 +14,20 @@ toc: true
 
 ## CoAP Gateway
 
-The CoAP gateway acts as a CoAP Client, communicating with IoT devices - CoAP Servers following the [OCF specification](https://openconnectivity.org/developer/specifications/). As the component diagram describes, the responsibilities of the gateway are:
+The CoAP gateway acts as a CoAP Client, enabling communication between IoT devices and CoAP Servers following the [OCF specification](https://openconnectivity.org/developer/specifications/). As the component diagram describes, the responsibilities of the gateway are:
```

Comment:
This is not correct. The device is a CoAP Server. The plgd CoAP Gateway is a CoAP Client. The CoAP Gateway is an entry point/gateway for devices, to the plgd hub.

Reply:
From the original I did think I might’ve rephrased it wrong. I guess the device is the thing giving information in response to requests, so that makes sense. 

Changing to:

“…communicating with IoT devices, which serve as CoAP Servers…”

(Unconcerned with repeating the word ’serve’ since I think it’s the right word in the context.)

Another possibility would be ‘…which are CoAP Servers’. But since ’server’ doesn’t really seem to define the device, I’ll suggest the former wording.

Reply:
P.S. I’ll come back to this after I’ve read a bit more about CoAP. Will probably change 'serve as'

Reply:
I like "which serve as CoAP Servers…" formulation.